js/client.js

```diff
@@ -1158,6 +1162,24 @@ function toId() {
 				break;
 			}
 		},
+		saveIgnore: function () {
+			var buf = {};
+			if (!this.ignore) return; // ??
+			for (var k in this.ignore) {
+				// we set this to false so they're notified once per session of it
+				buf[k] = 1;
+			}
+			Storage.prefs('ignorelist', buf);
```

Comment:
This should just be an array, right? It makes more sense as an array.

```suggestion
			if (!this.ignore) return; // ??
			Storage.prefs('ignorelist', Object.keys(this.ignore));
```